README.md

```diff
@@ -18,17 +18,17 @@ in a single repository and documented in a single set of
 [`solidus`](https://github.com/solidusio/solidus) gem you automatically require all
 of the necessary gem dependencies which are:
 
-* [`solidus_api`](https://github.com/solidusio/solidus/tree/master/api) (RESTful API)
-* [`solidus_frontend`](https://github.com/solidusio/solidus/tree/master/frontend) (Cart and storefront)
-* [`solidus_backend`](https://github.com/solidusio/solidus/tree/master/backend) (Admin area)
-* [`solidus_core`](https://github.com/solidusio/solidus/tree/master/core) (Essential models, mailers, and classes)
-* [`solidus_sample`](https://github.com/solidusio/solidus/tree/master/sample) (Sample data)
+* [`solidus_api`](/master) (RESTful API)
```

Comment:
I wouldn't use relative URLs here, because READMEs get read outside of GitHub as well and GitHub is smart enough the parse absolute links relatively.